fastlane_core/spec/command_executor_spec.rb

```diff
@@ -10,11 +15,13 @@
         expect(explodes_on_strip).to receive(:strip).and_raise(Errno::EIO)
 
         child_process_id = 1
-        expect(Process).to receive(:wait).with(child_process_id)
+        unless FastlaneCore::Helper.windows?
+          # expect(Process).to receive(:wait).with(child_process_id) // moved over to fastlane_pty.rb
```

Comment:
Can I somehow test the calling of `Process.wait()` here in its new place, where I moved it?

Reply:
Does this same `expect` not work here? I feel like it should 🤔 `Process` should still receive `:wait` - it shouldn't matter that it was moved into `fastlane_pty.rb`

Reply:
Maybe my logic of the move is wrong then? I couldn't get it to work yesterday :/

Reply:
Let me test (via CI).

Reply:
An idea: 

When `Process.wait` was in its old location, it was _always_ executed in this strange tests that tests for some edge case with EIO exception that I don't understand.

Now that I moved it, this doesn't happen any more when this exception is triggered because of the changed location.

Does that make sense?

Reply:
Ok, I finally understood: This EIO test doesn't _actually_ execute the `.spawn` but "mock" or "fake" it, that's why the `Process.wait` I moved into the code didn't get executed. Will update below.